scripts/generate_textbook.py

```diff
@@ -145,6 +146,22 @@ def _between_symbols(string, c1, c2):
     return string[string.index(c1)+1:string.index(c2)]
 
 
+def _case_sensitive_fs(path):
+    """ True when filesystem at `path` is case sensitive, False otherwise
```

Comment:
```suggestion
    """True when filesystem at `path` is case sensitive, False otherwise.
    
    Checks this by attempting to write two files, one w/ upper case, one
    with lower. If after this only one file exists, the system is case-insensitive.
```